jobs/opensearch/templates/config/opensearch-security/roles_mapping.yml.erb

```diff
@@ -49,3 +59,4 @@ kibana_server:
   - "kibanaserver"
   # Give permission for Dashboards to access Opensearch
   - "dashboard.opensearch.internal"
+  - "logstash.opensearch.internal"
```

Comment:
Does Logstash also need Kibana permissions?

Reply:
we can remove this